launchpad-guaranteed-tickets-v2/src/token_release.rs

```diff
@@ -5,11 +5,11 @@ use launchpad_common::{config, launch_stage};
 
 pub const MAX_PERCENTAGE: u64 = 10_000;
 pub const MAX_UNLOCK_MILESTONES_ENTRIES: usize = 60;
-pub const MAX_RELEASE_EPOCH_DIFF: u64 = 1800;
+pub const MAX_RELEASE_DIFF: u64 = 60 * 60 * 24 * 365 * 5;
```

Comment:
As a code reader, I have to think that this means... 5 years? Add a comment.

Reply:
Not really. It was 5 years when we used timestamps. Now, If we get to a 0.6s round duration, it will be less that 5 years. But at the same time, I don't want to specify the constant's value based on what we assume will be the block duration. Maybe make the multiply and put the value directly?

Reply:
+1

Reply:
Added a new, fixed value.